### PR TITLE
Fix up jshint. Fix #421

### LINKED ISSUE #421: Drag-drop in playlists refreshes page
Dragging and dropping a file from a playlist into itself (i.e. not into one of the BAPS channels) on Show Planner causes the page to refresh. This is probably not intended behaviour.

### PR DESCRIPTION
**Pull request by LloydW93**

### CODE REVIEW

File: src/Public/js/nipsweb.player.js [hunk 1]
```diff
@@ -111,8 +111,13 @@ var NIPSWeb = function(d) {
                         cache: false,
                         success: function(data) {
                             $('#notice').hide();
+<<<<<<< HEAD
+                            for (var i in data) {
+                                if (i === 'myury_errors') {
+=======
                             for (i in data) {
                                 if (i === 'myradio_errors') {
+>>>>>>> master
```
Comment (LordAro) [inline src/Public/js/nipsweb.player.js]: gj merging

File: src/Public/js/nipsweb.player.js [hunk 2]
```diff
@@ -208,12 +210,13 @@ var NIPSWeb = function(d) {
                         );
                         li.attr('timeslotitemid', 'findme');
 
-                    } else if (li.attr('channel') === 'res' || li.attr('channel') == null) {
+                    } else if ( $(this).attr('channel') !== 'res' && (li.attr('channel') === 'res' || li.attr('channel') == null)) {
```
Comment (AJDurant) [inline src/Public/js/nipsweb.player.js]: umm, what is this addition for?

File: src/Public/js/nipsweb.player.js [hunk 3]
```diff
@@ -244,21 +247,21 @@ var NIPSWeb = function(d) {
...
+                        */
                         var inc = [];
                         var dec = [];
 
                         $('ul.baps-channel li[channel=' + oldChannel + ']').each(
                             function() {
-                                if (oldWeight - $(this).attr('weight') < 0
-                                    && $(this).attr('timeslotitemid') !== li.attr('timeslotitemid')
+                                if (oldWeight - $(this).attr('weight') < 0 &&
+                                    $(this).attr('timeslotitemid') !== li.attr('timeslotitemid')
```
Comment (AJDurant) [inline src/Public/js/nipsweb.player.js]: but the previous one was following PSR

Comment (LloydW93) [inline src/Public/js/nipsweb.player.js]: PSR is for PHP...

File: src/Public/js/nipsweb.player.js (hunk 2 above)
Comment (LloydW93) [inline src/Public/js/nipsweb.player.js]: That's the bit that fixes the issue...

File: src/Public/js/nipsweb.player.js (hunk 1 above)
Comment (LloydW93) [inline src/Public/js/nipsweb.player.js]: ctrl+s is an important step of any merge.

File: src/Public/js/nipsweb.player.js (hunk 2 above)
Comment (AJDurant) [inline src/Public/js/nipsweb.player.js]: :+1:

**Pull request merged**